eng/referenceAssemblies.props

```diff
@@ -1,9 +1,9 @@
 <Project>
   <PropertyGroup Condition="'$(BuildAllProjects)' == 'true' and
-                            !$(BuildTargetFramework.StartsWith('netstandard'))">
-    <AdditionalBuildTargetFrameworks Condition="!$(BuildTargetFramework.StartsWith('net45'))">$(AdditionalBuildTargetFrameworks);netstandard2.0</AdditionalBuildTargetFrameworks>
-    <AdditionalBuildTargetFrameworks Condition="!$(BuildTargetFramework.StartsWith('netcoreapp2.0')) and
-                                                !$(BuildTargetFramework.StartsWith('net4'))">$(AdditionalBuildTargetFrameworks);netstandard2.1</AdditionalBuildTargetFrameworks>
+                            !$(BuildTargetFramework.StartsWith('netstandard')) and
+                            !$(BuildTargetFramework.StartsWith('net4'))">
+    <AdditionalBuildTargetFrameworks>$(AdditionalBuildTargetFrameworks);netstandard2.0</AdditionalBuildTargetFrameworks>
+    <AdditionalBuildTargetFrameworks Condition="!$(BuildTargetFramework.StartsWith('netcoreapp2.0'))">$(AdditionalBuildTargetFrameworks);netstandard2.1</AdditionalBuildTargetFrameworks>
```

Comment:
When would we ever specify `BuildTargetFramework` as `netcoreapp2.0`?  I can imagine cases where `TargetFramework` is this, but not `BuildTargetFramework`.  That implies someone would be trying to build a netcoreapp2.0 vertical from the latest codebase 😕

Reply:
> That implies someone would be trying to build a netcoreapp2.0 vertical from the latest codebase 😕

Right, you can already do that today. This change just fixes the over building of the reference assembly configurations for some .NETCoreApp tfms.

Reply:
It still doesn't make sense to me.  I don't understand the purpose of building a `netcoreapp2.0` vertical.  Additionally,  `netcoreapp2.0` doesn't even support `netstandard2.1`  why would we be building that when someone wants to build `netcoreapp2.0`

Reply:
> Additionally, netcoreapp2.0 doesn't even support netstandard2.1 why would we be building that when someone wants to build netcoreapp2.0

That's why the condition is inverted. We only add the netstandard2.1 ref assembly addition build target framework for everything .NETCoreApp except 2.0.

> It still doesn't make sense to me. I don't understand the purpose of building a netcoreapp2.0 vertical.

As said, this already works in master today and I'm only fixing the overbuilding here. If we want to stop supporting other BuildTargetFrameworks, let's do that in a follow-up PR.

Reply:
Never mind, this was me missing a `!`  sorry for the noise.